aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -131,7 +131,6 @@ pub enum FeatureFlag {
     FederatedKeyless,
     TransactionSimulationEnhancement,
     CollectionOwner,
-    TransactionContextHashFunctionUpdate,
```

Comment:
@georgemitenkov we are removing a feature flag, are you anywhere depending on index of EnableLoaderV2 directly? (as this change will change it's index)

Reply:
No, should be ok